### DISCUSSION TITLE
Quarto publish action with resources

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#7800] [Q&A]
### Description

When running `quarto render` locally with a yaml header like so:

```yaml
freeze: true
resources: 
  - html/lib
```

The html/lib gets carried over into `_site/`. However, when the quarto publish action is ran, the resources do not make it into the `gh-pages` branch. Is this because of the use of freeze? How can I get the resources to be included in the GHA?

**Comment by mcanouil** [comment]
Freeze option has nothing to do with resources, it's only regarding computations.
Note that from the little bit you shared, `resources` is not placed where it is documented to be (<https://quarto.org/docs/websites/website-tools.html#site-resources>).[^1]

Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

[^1]: SLightly related to #3766

**Reply by JosiahParry** [reply]
I can confirm that resources is placed correctly. I omitted the title field as i didn't think it was necessary for the sake of example. 

For a minimal reproducible example one would need a github repo configured with GHA. 

This related to this qmd file https://github.com/R-ArcGIS/r-bridge-site/blob/main/location-services/tutorials/shiny-dash/index.qmd
And the GHA produces the following file https://github.com/R-ArcGIS/r-bridge-site/tree/gh-pages/location-services/tutorials/shiny-dash/html

**Reply by mcanouil** [reply]
You don't need GitHub Actions to publish to GitHub Pages.
Why is it necessary to reproduce your issue? The local command should be enough.

**Reply by JosiahParry** [reply]
Alrighty then.

**Reply by JosiahParry** [reply]
It appears that this discussions repo is not a place to get technical assistance or feedback. But rather, receive opinions on how to do work in the open. Do you suggest an alternative place to address issues with Quarto?

**Reply by mcanouil** [reply]
That's not an opinion simply trying to rule out things that do not seem relevant to the issue.

I am asking if the issue is only from GitHub Actions workflow or if you also have it when publishing locally.

Note two things, first I am not a native English speaker thus my replies might not reflect tones or subtle elements of the language, second, this is not my day to day main job and don't have several hours to create small reproducible and remove all unnecessary parts to figure out what might be a Quarto bug or a user error.
If I choose to do this rather than asking user to do it, then users will have to wait longer and get answers not accurate or targeted to their use cases, because my example would be based on assumptions.

For short, help us help you.
